readthedocs/templates/projects/project_dashboard.html

```diff
@@ -1 +1,29 @@
 {% extends 'projects/project_dashboard_base.html' %}
+
+{% block dashboard-onboard-extra %}
+    <!-- BEGIN Gold Pitch -->
+    {% if not request.user.gold.count and request.user.projects.count > 1 %}
+      <div class="module onboard onboard-accounts">
+        <h3>Support Read the Docs</h3>
+
+        <p>
+            Read the Docs depends on users like you to help us keep the site sustainable.
+        </p>
+        <p>
+            We now have the concept of <a href="{% url "gold_detail" %}">Read the Docs Gold</a> to allow folks to support us. You can pay us a set amount every month, and we'll keep the site running, and improving all the time.
```

Comment:
This sounds awkward, this could instead mention gold "subscriptions" that "fund maintenance and support costs", and avoid being overly explicit with describing the transaction.